docs/standard/changelog.rst

```diff
@@ -6,25 +6,66 @@ ChangeLog
+* valid date formats for all date fields restricted 
+* Schema and codelist titles and descriptions updated
+* Documentation images and text updated 
+* Documentation guidance restructured into modelling requirements and system requirements
+* Ownership-or-control statements replaced with Relationship statements
+* Statement fields:
+  * ``statementDate`` now required
+  * ``statementType`` replaced with ``recordType``
+  * ``statementID`` renamed ``statementId``
+* Entity fields: 
+  * ``entityType`` and ``entitySubtype`` combined into ``entityType``
+  * ``securitiesListing`` removed from required fields in ``hasPublicListing``
+  * ``nomination`` and ``trust`` added to ``entitySubtype`` codelist
+  * prefixes removed from ``entitySubtype`` codelist (e.g. ``stateBody-stateAgency`` is now ``stateAgency``) 
+* Relationship fields:
+  * ``Interest/share/maximum`` is now inclusive by default. ``exclusiveMinimum`` and ``exclusiveMaximum`` are now numbers
+  * ``nominee`` and ``nominator`` added to interest type codelist 
+  * ``interestedParty.describedByEntityStatement``, ``interestedParty.describedByPersonStatement``, ``interestedParty.unspecified`` combined into ``interestedParty`` which is either a string referring to a recordId or an Unspecified Record. 
+  * ``componentStatementIds`` replaced with ``componentRecordIds``
+* Person fields:
+  * ``fullName`` is now required 
+  * Renamed nameType code ``individual`` to ``legal`` 
+* ``address.country`` is now a Country object not a country code. 
 
+Removed
+-------
+ * ``placeOfResidence``
+ * ``agent``
+ * ``replacesStatements``
+ * Functional requirements page
+ * Sources and annotations page
+
+ 
 Alterations to schema structure and logic
 -----------------------------------------
-- Updated the JSON Schema version from draft-04 to 2020-12.
-  - `Interest/share/maximum` is now inclusive by default. `exclusiveMinimum` and `exclusiveMaximum` are now numbers (instead of booleans), which are used in the same way as and alongside their inclusive counterparts `minimum` and `maximum`.
-  - `id` to `$id`
-  - `definitions` to `$defs`
-  - `enum`s with one value are now `const`
-- Updated which date formats are valid for all date fields.
-  - Interest `startDate` and `endDate`, PEP Status `startDate` and `endDate`, Entity `foundingDate` and `dissolutionDate`, and formedByStatute `date` are more strict - only date (YYYY-MM-DD) is valid; timestamps and partial dates are no longer valid.
-  - Person `birthDate` and `deathDate` are more strict - year (YYYY), year and month (YYYY-MM) and year, month and day (YYYY-MM-DD) are valid, but timestamps are no longer valid.
-  - Annotation `creationDate`, PublicationDetails `publicationDate` are less strict - valid as date or date-time (partial dates are still not valid).
+* Updated the JSON Schema version from draft-04 to 2020-12
+* Included as much validation as possible within the schema 
+* id to $id
```

Comment:
I think the id to $id and the def to $def change *may* be a byproduct of the move to JSON schema 2020-12. In which case we can remove these two lines.